templates/crane.conf.erb

```diff
@@ -5,5 +5,5 @@
 [general]
 debug: false
 #data_dir_polling_interval: 60
-data_dir: /var/www/pub/docker/app
+data_dir: /var/lib/pulp/published/docker/v2/app/
```

Comment:
Where does this come from? e.g. if I look at the default crane config I see https://github.com/pulp/crane/blob/master/crane/data/default_config.conf#L3 but I assume Pulp, when using crane, has a different config? Is that documented?